src/main/resources/mapper/person-mapper.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper
+        PUBLIC "-//mybatis.org//DTD Config 3.0//EN"
+        "https://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="com.solvd.app.interfaces.IPersonDAO">
+    <resultMap id="personResultMap" type="Person" autoMapping="false">
+        <id property="personID" column="person_id"/>
+        <result property="name" column="name"/>
+        <result property="address" column="address"/>
+        <result property="phoneNumber" column="phone_number"/>
+        <result property="email" column="email"/>
+        <result property="gender" column="gender"/>
+    </resultMap>
+
+    <insert id="createEntity" keyProperty="person_id">
+        INSERT INTO people (name, address, phone_number, email, gender)
+        VALUES (#{name}, #{address}, #{phoneNumber}, #{email}, #{gender});
+    </insert>
```

Comment:
how is the personId returned from database to use it to add in CustomerService.createEntity?

Reply:
Hi, I realized that I forgot to put "useGeneratedKeys="true"". I just fixed it. Thanks for pointing it out.